builtin/providers/google/resource_compute_disk.go

```diff
@@ -185,6 +186,28 @@ func resourceComputeDiskCreate(d *schema.ResourceData, meta interface{}) error {
 	return resourceComputeDiskRead(d, meta)
 }
 
+func resourceComputeDiskUpdate(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	project, err := getProject(d, config)
+	if err != nil {
+		return err
+	}
+
+	if d.HasChange("size") {
```

Comment:
Does the machine have to be stopped? Do we have to wait for a specific status before we know that this has worked as expected?